### PR TITLE
Added libxml2.dev to debian/control

### PR DESCRIPTION
**Pull request by OlTwello**
Added libxml2-dev

### CODE REVIEW

**Comment by basvandervlies** [comment]
+1. You can merge it

**Comment by OlTwello** [comment]
Not a branch, so rejected

File: debian/control [hunk 1]
```diff
@@ -4,7 +4,7 @@ Priority: optional
 Maintainer: Antonio Radici <antonio@debian.org>
 Build-Depends: debhelper (>= 7.0.50), autotools-dev, libssl-dev,
  flex, bison, libpcre3-dev, dh-autoreconf,  libtokyocabinet-dev, libvirt-dev, libacl1-dev,
-  libmysqlclient-dev, libpq-dev
+  libmysqlclient-dev, libpq-dev, libxml2.dev
```
Comment (jooooooon) [inline debian/control:7]: I think this should be `libxml2-dev`.

Comment (Webhuis) [inline debian/control:7]: Dear Jonathan,

> > @@ -4,7 +4,7 @@ Priority: optional
> >  Maintainer: Antonio Radici antonio@debian.org
> >  Build-Depends: debhelper (>= 7.0.50), autotools-dev, libssl-dev,
> >   flex, bison, libpcre3-dev, dh-autoreconf,  libtokyocabinet-dev,
> > libvirt-dev, libacl1-dev,
> > -  libmysqlclient-dev, libpq-dev
> > -  libmysqlclient-dev, libpq-dev, libxml2.dev
> 
> I think this should be `libxml2-dev`.

You are quite right. Please see:
https://github.com/Webhuis/Cfengine-debian/issues/13

Met vriendelijke groet,
Martin.

+31651567029

<html><body><a href="http://nl.linkedin.com/pub/martin-simons/1/b78/966"
style="text-decoration:none;"><span style="font: 80% Arial,sans-serif;
color:#0783B6;">View Martin Simons's LinkedIn profileView Martin Simons's
profile</span></a></body></html>

**Pull request closed without merging**